js/src/common/components/Button.tsx

```diff
@@ -107,7 +116,7 @@ export default class Button<CustomAttrs extends IButtonAttrs = IButtonAttrs> ext
 
     const { 'aria-label': ariaLabel } = this.attrs;
 
-    if (!ariaLabel && !extractText(vnode.children) && !this.element?.getAttribute?.('aria-label')) {
```

Comment:
Instead of doing this, could we check whether the view method is overriden?

[Playground Link](https://www.typescriptlang.org/play?#code/MYGwhgzhAECC0G8BQ1XQGYFcB2wAUAlIimqQE4CmALpmdtAORgMDcJqAvku9AJYQAxHMADyANwpkyvACYyK2QsVLlqtelQAW-AHRZc0AIQBeY3B0AHMgHsqtgJ4WKe4Ty5ckoSDABC0CgAeVAoyMPDIpPr4RBEqqJQ0dIwARqxuSB7A1tgQ1iDOINYA5nh42BQA7nCEBDr8QrjiktJyCjWe2bn5OoUlZZXQPjV1gsJNUrLyigQEQA)